prow/config/prow-config-documented.yaml

```diff
@@ -293,6 +293,60 @@ deck:
         # Logo is the location of the logo that will be loaded in deck.
         logo: ' '
 
+    # DefaultRerunAuthConfigs is a list of DefaultRerunAuthConfigEntry structures that specify who can
+    # trigger job reruns. Reruns are based on whether the entry's org/repo or cluster matches with the
+    # expected fields in the given configuration.
+
+    # Each entry in the slice specifies Repo and Cluster regexp filter fields to
+    # match against jobs and a corresponding RerunAuthConfig. The entry matching the job with the
+    # most specification is for authentication purposes.
+
+    # This field is smarter than the RerunAuthConfigs, because each
+    # entry includes additional Cluster regexp information that the old format
+    # does not consider.
```

Comment:
This is the key piece of information that explains this PR. It also explains why you need to use a slice (why a map won't work). Can you please highlight this in your commit message?

Reply:
Fixed!